File: Examples/CSharp/NoteBook/CreateNoteBook.cs
using System.IO;
using Aspose.Note;
using System;
using System.Collections.Generic;

namespace Aspose.Note.Examples.CSharp.WorkingWithNoteBook
{
    public class CreateNoteBook
    {
        public static void Run()
        {
            // ExStart:CreateNoteBook
            // ExFor:Notebook
            // ExSummary:Shows how to save notebook.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_NoteBook();
            
            var notebook = new Notebook();

            dataDir = dataDir + "test_out.onetoc2";

            // Save the Notebook
            notebook.Save(dataDir);

            // ExEnd:CreateNoteBook

            Console.WriteLine("\nNoteBook created successfully.\nFile saved at " + dataDir);
        }
    }
}

File: Examples/VisualBasic/NoteBook/CreateNoteBook.vb
Imports System.IO
Imports Aspose.Note
Imports System.Collections.Generic

Namespace WorkingWithNoteBook
    Public Class CreateNoteBook
        Public Shared Sub Run()
            ' ExStart:CreateNoteBook
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_NoteBook()

            Dim notebook = New Notebook()

            dataDir = dataDir & Convert.ToString("test_out.onetoc2")

            ' Save the Notebook
            notebook.Save(dataDir)

            ' ExEnd:CreateNoteBook
            Console.WriteLine(Convert.ToString(vbLf & "NoteBook created successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace